.github/workflows/blueskyfeedbot.yml

```
@@ -0,0 +1,36 @@
+name: FeedBot
+on:
+  schedule:
+    # This will run every 10 minutes
+    - cron: '*/10 * * * *'
```

Not sure that we really have enough blog posts to warrant this? Every hour wold be fine?

```suggestion
    # This will run every hour
    - cron: '0 * * * *'
```